CFDP/Checksum/Checksum.hpp

```diff
@@ -17,9 +17,40 @@
 
 namespace CFDP {
 
-  //! \class Checksum
-  //! \brief Class representing a CFDP checksum
-  //!
+  /*! \class Checksum
```

Comment:
I see two different styles in this file for block comments:

```
/*!
 *
 */
```

And:

```
//!
//!
//!
```
Can we converge on one?  F Prime is converging on the second one (`//!`)

Other than that, this looks good!

Reply:
I switched the class comment to line comment style.